pkg/kubectl/resource/builder.go

```diff
@@ -656,6 +671,9 @@ func (b *Builder) visitorResult() *Result {
 // inputs are consumed by the first execution - use Infos() or Object() on the Result to capture a list
 // for further iteration.
 func (b *Builder) Do() *Result {
+	for ix := range b.paths {
```

Comment:
That requires that the transformer be set on the Builder object before the paths are set.   This is fairly non-intuitive for the user, for example, I could do:

```
builder.Stdin().SetTransform(...).Files(...)
```

and the transform would be applied to the files but not to stdin.

We could panic, I suppose, but that seems far uglier than having the setter.